lib/eibkolibri.py

```diff
@@ -29,6 +30,12 @@
 
 class RemoteKolibri:
     """Kolibri remote instance"""
+
+    class Series(enum.Enum):
+        """Supported Kolibri server series"""
+        KOLIBRI_0_15 = enum.auto()
+        KOLIBRI_0_16 = enum.auto()
```

Comment:
A chaotic alternative would be:

```
        KOLIBRI_0_15 = 0.15
        KOLIBRI_0_16 = 0.16
```

I thought this would be rejected but actually it's completely legal and the Python documentation gives an example where enum values are tuples of floats.

https://docs.python.org/3/howto/enum.html#planet